### ISSUE TITLE
Extract resolvers into a separate package so they can be used in other applications 

### ISSUE THREAD

**Issue by Stvad** [fuddl/wd#39]
Example use-case: https://twitter.com/VladyslavSitalo/status/1462908339860430855

### Considesations:
- can keep code in the same repo (making this a monorepo) potentially simplifying code management initially but potentially making future contributions harder
  - Or extract into a separate repo
- License: using a more permissive license (MIT, Apache, etc) is IMO a better approach for the library
- It'd be nice to have a single interface for accessing the resolvers (e.g. a set of functions) (vs current situation where users have to iterate through all resolvers)
- Depends on #34
